src/material-experimental/mdc-slide-toggle/_slide-toggle-theme.scss

```diff
@@ -1,5 +1,5 @@
 @use '@material/theme/theme-color' as mdc-theme-color;
-@use '@material/switch' as mdc-switch;
+@use '@material/switch/deprecated' as mdc-switch;
```

Comment:
When we switched the MDC list to the deprecated mixins, we also had to rename some CSS classes. Is that the case here too?